oozie-to-airflow/mappers/base_mapper.py

```diff
@@ -69,6 +69,20 @@ def last_task_id(self) -> str:
         """
         return self.name
 
+    def on_parse_node(self, workflow):
```

Comment:
Why do we need `workflow` in the signature? The only 2 usages of this method don't need it.
Don't you think we should remove this for now and add if needed in the future?

Reply:
I think it's good to leave it as it is. It will be uniform with the second method.

Reply:
I don't know, if it's not needed then why use it? What do you think, @potiuk?

Reply:
In fact I think the right way of doing it is to move archives and files to workflow and then we will use workflow as parameter. Then it will be consistent with our approach where parsed data is in the workflow now.

Reply:
I look at it now more closely and I am not sure any more. I am not sure yet how it should be yet, but I have a feeling we spend too much time on it already. Let me think about it.